truffle/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "truffle",
+  "version": "1.0.0",
+  "description": "",
+  "main": "truffle-config.js",
+  "directories": {
+    "test": "test"
+  },
```

Comment:
I don't think `"directories"` does anything here? If so let's get it outta here.